novawallet/Common/Helpers/ChainsStore.swift

```diff
@@ -51,6 +61,10 @@ extension ChainsStore: ChainsStoreProtocol {
     }
 
     func getChain(for chainId: ChainModel.Id) -> ChainModel? {
-        chainRegistry.getChain(for: chainId)
+        let chain = chainRegistry.getChain(for: chainId)
```

Comment:
As we agreed let's also filter available chains to be consistent. So, I suggest filter availableChainIds applying filter to the corresponding chain (chainRegistry.getChain(...))

Reply:
Thanks, done